ext/arrow-nmatrix/extconf.rb

```diff
@@ -14,7 +14,7 @@
 
 require "mkmf-gnome2"
 
-required_pkg_config_package("arrow-glib")
+required_pkg_config_package(["arrow-glib", 3])
```

Comment:
I think we don't need to support old arrow-glib.

Reply:
Ah, we also need to update `.gemspec`.

Reply:
Oops...